package.json

```diff
@@ -130,8 +132,18 @@
         "category": "OpenShift"
       },
       {
-        "command": "openshift.component.create",
```

Comment:
let's keep this command and make it visible only in context menu for Application node.